sql/src/test/java/org/apache/druid/sql/calcite/DecoupledPlanningCalciteQueryTest.java

```diff
@@ -337,5 +337,25 @@ public void testFilterOnCurrentTimestampWithIntervalArithmetic()
   public void testFilterOnCurrentTimestampOnView()
   {
 
+  }
+  @Override
+  @Ignore
+  public void testExactCountDistinctWithGroupingAndOtherAggregators()
+  {
+
+  }
```

Comment:
did these previously work with the decoupled planner and no longer do?

Reply:
It's a weird case that when run through the Decoupled planner the order of the dimensions changes in setDimensions. The rest of the plan remains the same. This has been ignored for that reason, I'll put a comment to revisit this